### PR TITLE
Added O2R Support

### PR DESCRIPTION
**Pull request by KiritoDv**

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "yaml.schemaStore.enable": false
```
Comment (briaguya-ai) [inline .vscode/settings.json:2]: we should probably `.gitignore` the `.vscode` dir

File: lib/arc/arc.dart [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+import 'package:archive/archive.dart';
+import 'package:flutter_storm/flutter_storm.dart';
+import 'package:flutter_storm/flutter_storm_defines.dart';
+import 'package:retro/utils/log.dart';
+
+enum ArcMode {
+  otr, o2r
+}
+
+class Arc {
```
Comment (briaguya-ai) [inline lib/arc/arc.dart:16]: nitpick(naming): i'd prefer `Archive` to `Arc` but that shouldn't block merging

File: lib/features/create/create_finish/create_finish_screen.dart [hunk 3]
```diff
@@ -128,7 +128,7 @@ class _CreateFinishBottomBarModalState
                                   )
                                 ],
                               )
-                            : Text(i18n.createFinishScreen_generateOtr),
+                            : Text(i18n.createFinishScreen_generateOtr, style: TextStyle(color: Colors.white)),
```
Comment (briaguya-ai) [inline lib/features/create/create_finish/create_finish_screen.dart:131]: just curious: why did this need to happen?

File: test/widget_test.dart [hunk 4]
```diff
@@ -12,7 +12,7 @@ import 'package:flutter_test/flutter_test.dart';
 void main() {
   testWidgets('Counter increments smoke test', (WidgetTester tester) async {
     // Build our app and trigger a frame.
-    await tester.pumpWidget(const MyApp());
+    // await tester.pumpWidget(const MyApp());
```
Comment (briaguya-ai) [inline test/widget_test.dart:15]: do we need this test at all? can we just remove all of `testWidgets`?

**Review by briaguya-ai — COMMENTED** [review]
overall looking great! couple little comments

File: lib/features/create/create_finish/create_finish_screen.dart (hunk 3 above)
Comment (skacontreras) [reply lib/features/create/create_finish/create_finish_screen.dart:131]: Flutter fault moving to material 3

File: lib/arc/arc.dart (hunk 2 above)
Comment (KiritoDv) [reply lib/arc/arc.dart:16]: Im gonna think about it, but i love Arc tbh xD

**Review by briaguya-ai — APPROVED** [review]

**Pull request merged**